index.js

```diff
@@ -1,18 +1,19 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1
+console.log(counter) 
+counter = 2
 
+console.log("%c----------", "color: red") 
+console.log(counter) 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
-
+// const name = "Richard"
```

Comment:
Great! The key here was using `const` because it doesn't allow us to reassign our variable